### DISCUSSION TITLE
I can't figure out the most basic form

### DISCUSSION THREAD

**Discussion by abolotnov-ion** [Budibase/budibase#5330] [Help] [answered]
I have a REST API that only accepts POST requests and I am trying to build a form that would surrender data to it. I have registered the api and its fields:

<img width="732" alt="image" src="https://user-images.githubusercontent.com/83080383/162549181-69f0d072-bed6-4e25-9afe-a6a26b88e261.png">

And then added a screen with a form:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/83080383/162549220-fdae8c11-0231-4716-a2de-c0ce7625159a.png">

I pointed form's schema to the said API and added a field group (I though it would pick up params from body) but it's not. I can add the fields manually but I can't figure out how to bind them to the POST body params. Do I also need to add the buttons manually?

**Comment by abolotnov-ion** [comment]
Never mind, could not figure out how to delete it, but I moved on to a different platform.

**Reply by arladmin** [reply]
May i ask, which platform did you move on to?

**Reply by abolotnov-ion** [reply]
appSmith

**Comment by Rory-Powell** [comment, marked as answer]
Hey @abolotnov-ion. Thanks for the question, we have a guide for exactly this. Check it out here: https://docs.budibase.com/docs/rest-bindings
